features/org.wso2.carbon.deployment.engine.feature/pom.xml

```diff
@@ -81,6 +84,32 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.wso2.carbon.deployment</groupId>
+                                    <artifactId>org.wso2.carbon.deployment.engine</artifactId>
+                                    <version>${carbon.deployment.version}</version>
+                                    <type>bundle</type>
+                                    <overWrite>true</overWrite>
+                                    <outputDirectory>${project.build.directory}/docs</outputDirectory>
+                                    <includes>config-docs/**</includes>
```

Comment:
Please raise a github issue in carbon-kernel repo where 'org.wso2.carbon: org.wso2.carbon.extensions.configuration.maven.plugin' artifact is maintained to add the config-docs inside target/classes folder.

The rationale is, the contents under src/main/resources already goes to target/classes/ folder. The config-docs is another resource file that should be packed within the bundle itself.